spec/presenters/school_vacancies_presenter_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'rails_helper'
+RSpec.describe SchoolVacanciesPresenter do
+  let(:school) { create(:school) }
+  let(:presenter) { described_class.new(school, 'published') }
+
+  let!(:draft_vacancies) { create_list(:vacancy, 5, :draft, school: school) }
+  let!(:pending_vacancies) { create_list(:vacancy, 2, :future_publish, school: school) }
+  let!(:expired_vacancies) do
+    expired_vacancies = []
+    3.times do
+      expired_vacancy = build(:vacancy, :expired, school: school)
+      expired_vacancy.save(validate: false)
+      expired_vacancies << expired_vacancy
+    end
+    expired_vacancies
+  end
+  let!(:live_vacancies) { create_list(:vacancy, 7, :published, school: school) }
+
+  describe 'initialize' do
+    it 'raises an error if the type is not recognised' do
+      expect { SchoolVacanciesPresenter.new(school, 'something') }.to raise_error(ArgumentError)
+    end
+  end
+
+  it 'returns the vacancy type' do
```

Comment:
The convention of wrapping methods in a `describe` block is useful as it makes it easier for devs as a grouping method so we can find all the relevant tests to a method and we'll also be able to add 'contexts' and 'before' steps accurately and only affect the relevant tests. Without this, it is all too easy for methods to get added out of order or moved around in other refactoring.

If we start off the file with describe blocks even when we don't obviously need them right now we set the precedent that the next contributor can follow to ensure the file grows in a more manageable way. We remove the need for them to question "why was this done and should I refactor this first?" which I see very much as value.